### PR TITLE
Update k8s auth docs for new parameter

### PR DESCRIPTION
**Pull request by tvoran**
Adds info about the disable_local_ca_jwt parameter added in https://github.com/hashicorp/vault-plugin-auth-kubernetes/pull/97

To be deployed with 1.5.4.

### CODE REVIEW

File: website/pages/api-docs/auth/kubernetes/index.mdx [hunk 1]
```diff
@@ -22,6 +22,11 @@ existence with the Kubernetes TokenReview API. This endpoint configures the
 public key used to validate the JWT signature and the necessary information to
 access the Kubernetes API.
 
+If they aren't specified, Vault will use the local CA cert and service account
```
Comment (catsby) [inline website/pages/api-docs/auth/kubernetes/index.mdx:25]:
```suggestion
By default Vault will use the local CA cert and service account
```

File: website/pages/api-docs/auth/kubernetes/index.mdx [hunk 2]
```diff
@@ -22,6 +22,11 @@ existence with the Kubernetes TokenReview API. This endpoint configures the
...
+JWT for the `kubernetes_ca_cert` and `token_reviewer_jwt` parameters,
+respectively, when running in a Kubernetes pod. This behavior may be disabled by
+setting `disable_local_ca_jwt` to `true`.
```
Comment (catsby) [inline website/pages/api-docs/auth/kubernetes/index.mdx:28]: If users set this to true do they then also need to specify `kubernetes_ca_cert` and `token_reviewer_jwt` as well?

**Review by catsby — COMMENTED** [review]
I made a suggestion and ask a clarifying question. I believed we discussed this last week so please forgive me if I'm asking you to repeat yourself, I just remember this slightly differently than the documentation says here

File: website/pages/api-docs/auth/kubernetes/index.mdx (hunk 2 above)
Comment (tvoran) [reply website/pages/api-docs/auth/kubernetes/index.mdx:28]: If they set `disable_local_ca_jwt` to `true`, then either `kubernetes_ca_cert` or `pem_keys` needs to be specified as well. `token_reviewer_jwt` is optional.

Comment (tvoran) [reply website/pages/api-docs/auth/kubernetes/index.mdx]: I'm not sure if it helped, but I tried to elaborate a little more in 7fe90f0.

File: website/pages/api-docs/auth/kubernetes/index.mdx [hunk 3]
```diff
@@ -41,7 +41,17 @@ access the Kubernetes API.
 - `issuer` `(string: "")` - Optional JWT issuer. If no issuer is specified, then this plugin will
   use `kubernetes.io/serviceaccount` as the default issuer.
 - `disable_iss_validation` `(bool: false)` - Disable JWT issuer validation. Allows to skip ISS validation.
+- `disable_local_ca_jwt` `(bool: false)` - Disable defaulting to the local CA cert and service account JWT when running in a Kubernetes pod.
 
+### Caveats
+
+Either `kubernetes_ca_cert` or `pem_keys` must be set.
```
Comment (calvn) [inline website/pages/api-docs/auth/kubernetes/index.mdx:48]: This is dependent on the value from `disable_local_ca_jwt`?

**Review by calvn — APPROVED** [review]

File: website/pages/api-docs/auth/kubernetes/index.mdx (hunk 3 above)
Comment (tvoran) [reply website/pages/api-docs/auth/kubernetes/index.mdx]: Good question, tried to clarify in 6588f33 and e6d098f.

**Pull request merged**